### DISCUSSION TITLE
graphql/envelop/plugins/use-sentry

### DISCUSSION THREAD

**Discussion by giscus** [dotansimha/envelop#2075] [Docs Discussions]
# graphql/envelop/plugins/use-sentry

GraphQL Envelop Documentation

https://the-guild.dev/graphql/envelop/plugins/use-sentry

**Comment by gajus** [comment]
Appears as if there is no way to use this plugin without adding @sentry/tracing?

**Comment by FezVrasta** [comment]
I'm getting an error because the transactions seem to be not supported anymore on recent Sentry versions?
